scripts/install_third_party_libs_test.py

```diff
@@ -62,20 +63,52 @@ def mock_print(msg, end=''):
             subprocess, 'check_call', mock_check_call)
         self.print_swap = self.swap(python_utils, 'PRINT', mock_print)
 
+    def test_tweak_yarn_executable(self):
+        # pylint: disable=unused-argument
+        def mock_is_file(filename):
```

Comment:
If you just use unused_filename as the arg, you don't need the pylint disable, right?

Reply:
Did not notice this also works. Updated